src/components/callout/callout.helpers.ts

```diff
@@ -0,0 +1,40 @@
+import translate from '../../global/translations';
+
+const calloutIcons: Record<string, string> = {
+    danger: 'poison',
+    info: 'info',
+    note: 'appointment_reminders',
+    tip: 'idea',
+    warning: 'error',
+    success: 'ok',
+    failure: 'high_priority',
+    example: 'test_tube',
+    quote: 'quote_left',
+    question: 'help',
+};
+
+export function getIcon(icon: string, type: string = 'info'): string {
+    if (icon) {
+        return icon;
+    }
+
+    return calloutIcons[type] ?? calloutIcons.info;
+}
+
+export function getHeading(
+    heading: string,
+    type: string = 'info',
+    language: string = 'en'
+): string {
+    if (heading) {
+        return heading;
+    }
+
+    const key = `callout.${type}`;
+
+    try {
+        return translate.get(key, language);
+    } catch (error) {
+        return translate.get(key, 'en');
+    }
```

Comment:
@TommyLindh2 Don't forget to write about why you did this hack, and that we think this is not the place to do this hack.

From what I remember, if someone chose for example `fr`, `de`, or `nb` from the dropdown, the example crashed. Because `fr.po` and or `nb.po` files do no exist in Lime Elements, but the list of languages states that they are supported.